### ISSUE TITLE
Checkout API missing PaymentResponseAction in PaymentDetailsResponse model

### ISSUE THREAD

**Issue by franzmango** [Adyen/adyen-go-api-library#245]
**Describe the bug**
When doing a PaymentDetails request in response to IdentifyShopper flow that returns a ChallengeShopper result code, there is no data returned concerning the further steps to be taken.

If I read the the raw HTTP response body, the 'action' object exists, but it does not in the PaymentDetailsResponse model. The response field is also not documented in the API Explorer.

**Expected behavior**
I would expect it to be there, to be able to carry on with the 3DS flow 😄

**Additional context**
I am following this guide to implementing 3DS2 into our system, and it took me a while to figure out what I'm doing wrong since the Payment Details response had just the result code.

**Comment by michaelpaul** [comment]
Hello @franzmango, the library must contain the same fields as API explorer. Sometimes new fields get documented before even appearing on API explorer or libraries. Are following some guide on our docs? Could you please share a link to it and which step you got stuck?

**Comment by franzmango** [comment]
I was following this guide:
https://docs.adyen.com/checkout-build-your-own-3ds2

And I was stuck at the end of the 3DS2 fingerprint chapter:
https://docs.adyen.com/checkout-build-your-own-3ds2/#get-the-3d-secure-2-device-fingerprint

It presented me with the ChallengeShopper result code, but I could not figure out from this guide from where to get next challengeToken to continue with the 3DS2 flow.

Basically, at the moment I use a workaround getting the challengeToken from the /payments/details HTTP response body.

**Comment by michaelpaul** [comment]
Thanks for the clarification @franzmango. The guide you are following is outdated, it was written for Checkout v66. The library is based on the latest version (v70 at the moment). That's why you can't find the fields you expected.

Please consider using the newer versions of our [3DS2 guides](https://docs.adyen.com/online-payments/3d-secure/). For example [here](https://docs.adyen.com/online-payments/3d-secure/native-3ds2/web-component/?tab=v67-submit_1), you can see that `action` is no longer necessary and was in fact [removed since v67](https://docs.adyen.com/online-payments/release-notes/#releaseNote=2021-03-01-checkout-api-67).

**Comment by franzmango** [comment]
My bad, thanks!

One more question though, if I already have the older flow implemented, is it a fluke that it works with the v70 test endpoints, or is it some backward compatibility thing where the flow is still supported until some version?
